### PR TITLE
Adding more logs and formatting

### PR DESCRIPTION
**Pull request by anushreeringne**
Adding more logging for instance as below:

Host Rack Information:
Broker: 1003 Rack: FD2UD2
Broker: 1004 Rack: FD2UD3
Broker: 1009 Rack: FD0UD0
Broker: 1005 Rack: FD1UD4
Broker: 1001 Rack: FD1UD1
Broker: 1008 Rack: FD0UD1
Broker: 1007 Rack: FD0UD3
Broker: 1006 Rack: FD2UD0
Broker: 1002 Rack: FD1UD2

2018-04-05 05:10:07,396 - rebalance_rackaware.py [120111] __main__ - INFO - Topic: thu1 Partitions Already Balanced: [4] Partitions To Be Rebalanced: [0, 1, 2, 3, 5, 6, 7, 8, 9]
2018-04-05 05:10:07,396 - rebalance_rackaware.py [120111] __main__ - INFO -

Reassignment changes: Topic: thu1
Partition 0 [1001, 1002, 1003] --> [1002, 1004, 1009]
Partition 1 [1002, 1003, 1004] --> [1001, 1003, 1007]
Partition 2 [1003, 1004, 1005] --> [1005, 1006, 1008]
Partition 3 [1004, 1005, 1006] --> [1004, 1002, 1009]

### CODE REVIEW

**Comment by anushreeringne** [comment]
@ravitandonrt  @metafractal   can you please review?

File: src/python/rebalance/rebalance_rackaware.py [hunk 1]
```diff
@@ -444,7 +439,8 @@ def generate_reassignment_plan(plan_directory, topics, brokers_info, compute_sto
     if compute_storage_cost:
         partitions_sizes = get_storage_info(host_info)
         logger.debug("Partition Sizes: %s", str(partitions_sizes))
-    logger.debug("Host Information: %s", str(host_info))
+    logger.debug("\nHost Information: \n%s", host_info)
+    logger.info("\n\nHost Rack Information: \n%s\n", "\n".join(list(map(lambda datum: "Broker: {0} Rack: {1}".format(str(datum['brokerId']), datum['rack']), host_info))))
```
Comment (duoxu) [inline src/python/rebalance/rebalance_rackaware.py]: Should we use debug instead of info here?

File: src/python/rebalance/rebalance_rackaware.py [hunk 2]
```diff
@@ -490,29 +486,34 @@ def generate_reassignment_plan(plan_directory, topics, brokers_info, compute_sto
 
             logger.debug("Start with position in Rack Alternated List: %s", str(next_Leader))
 
-            reassignment_plan, balanced_partitions_for_topic = rassignment_Generator._generate_reassignment_plan_for_topic(replica_count_topic, next_Leader, rack_alternated_list, fd_count, ud_count, brokers_replica_count, force_rebalance)
-            logger.debug("Already balanced partitions for the topic %s are: %s", topic, balanced_partitions_for_topic)
+            reassignment_plan, balanced_partitions_for_topic = reassignment_generator._generate_reassignment_plan_for_topic(replica_count_topic, next_Leader, rack_alternated_list, fd_count, ud_count, brokers_replica_count, force_rebalance)
+            logger.debug("\nAlready balanced partitions for the topic %s are: \n%s", topic, balanced_partitions_for_topic)
```
Comment (duoxu) [inline src/python/rebalance/rebalance_rackaware.py:490]: Is "\n" at the beginning of a log message intented?

File: src/python/rebalance/rebalance_rackaware.py [hunk 3]
```diff
@@ -490,29 +486,34 @@ def generate_reassignment_plan(plan_directory, topics, brokers_info, compute_sto
...
 
             for partition in balanced_partitions_for_topic:
                 global_balanced_partitions.append(partition)
             if reassignment_plan is not None:
-                logger.info("Generated reassignment plan for topic: %s", topic)
+                logger.info("\n\nReassignment plan generated for topic: %s\n%s\n", topic, reassignment_plan)
```
Comment (duoxu) [inline src/python/rebalance/rebalance_rackaware.py:495]: The same here. Is "\n\n" intented? Every log message should have started on a new line.

File: src/python/rebalance/rebalance_rackaware.py (hunk 2 above)
Comment (anushreeringne) [reply src/python/rebalance/rebalance_rackaware.py:490]: yes it was intentional

File: src/python/rebalance/rebalance_rackaware.py (hunk 3 above)
Comment (anushreeringne) [reply src/python/rebalance/rebalance_rackaware.py:495]: yup. These are important logs hence wanted to add the \n here

File: src/python/rebalance/rebalance_rackaware.py (hunk 1 above)
Comment (anushreeringne) [reply src/python/rebalance/rebalance_rackaware.py]: changed

**Pull request merged**